src/status_im/ui/screens/db.cljs

```diff
@@ -69,7 +72,7 @@
 (spec/def ::network-status (spec/nilable keyword?))
 
 (spec/def ::mailserver-status (spec/nilable keyword?))
-(spec/def ::peers-count (spec/nilable integer?))
+(spec/def :inbox/topics set?)
```

Comment:
<img width="550" alt="screenshot 2018-05-21 14 10 25" src="https://user-images.githubusercontent.com/23836/40304802-bd3e9496-5d00-11e8-894f-9a4b6ace64ac.png">

I have spec failed on app start for `:inbox/topics` `set?` check.

Reply:
are you sure you started in  a clean state ? I didn't get this error in all my of my testing. Do you have steps to reproduce ?

Reply:
@yenda something was wrong with my state, cleaned and rebuild, errors gone.